doc/whats_new/v0.20.rst

```diff
@@ -22,8 +22,6 @@ random sampling procedures.
 
 - :class:`decomposition.IncrementalPCA` in Python 2 (bug fix)
 - :class:`isotonic.IsotonicRegression` (bug fix)
-- :class:`linear_model.ARDRegression` (bug fix)
```

Comment:
You seem to have removed some text from v0.20.rst, probably without realising. Please look at your diff and re-add the text you remove.